toolchain/check/scope_stack.h

```diff
@@ -159,7 +169,7 @@ class ScopeStack {
 
     // Names which are registered with lexical_lookup_, and will need to be
     // unregistered when the scope ends.
-    llvm::DenseSet<SemIR::NameId> names;
+    llvm::DenseSet<SemIR::NameId> names = {};
```

Comment:
Out of curiosity, why this addition? Some warning?

Reply:
Yes, this fixes a clang-tidy missing initializer warning.